.github/actions/compute-sdk-test/main.js

```diff
@@ -1,13 +1,12 @@
 // Node & 3P Modules
-const fs = require('fs');
-const path = require('path');
-const childProcess = require('child_process');
-const fetch = require('node-fetch');
-const chalk = require('chalk');
-require('better-logging')(console, {
-  format: ctx => {
-
-    const tag = chalk.bold(`[compute-sdk-test]`); 
+const fs = require("fs");
+const path = require("path");
+const childProcess = require("child_process");
+const fetch = require("node-fetch");
+const chalk = require("chalk");
+require("better-logging")(console, {
```

Comment:
If we can replace these dependencies with nodejs built-in packages such as `http`, then we will no longer need to have a build step nor need to commit the `dist` directory as we could run this file directly with GitHub Actions

Reply:
I'm all for it!

Reply:
I'm all for it!